.gitignore

```diff
@@ -31,6 +31,7 @@ node_modules
 /.sass-cache
 /connect.lock
 /coverage
+/examples/react-todos-app/coverage
```

Comment:
```suggestion
**/coverage
```

Reply:
this would exclude all coverage folders in the repo, similar to `**/.code-pushup`

Reply:
All our packages put their results in `/coverage` except for the `react-todos-app` special "nx project" so I don't think it is necessary to make it so generic.